src/07_slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[slice(1, 2)])
 
```

Comment:
Nice work here. Please note that the same can be accomplished with `a[1]` and similar syntax with the rest of these in the file.